chat.koishi.koishi-desktop.yml

```diff
@@ -0,0 +1,105 @@
+id: chat.koishi.koishi-desktop
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+  - org.freedesktop.Sdk.Extension.node22
+command: koi
+rename-appdata-file: chat.koishi.desktop.appdata.xml
+rename-desktop-file: chat.koishi.desktop.desktop
+rename-icon: koishi-app
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+  - --filesystem=home
```

Comment:
what is home for?

Reply:
The `~/.koishi` directory is where Koishi stores its configuration files.

Reply:
Use persist